scripts/zones/Yhoator_Jungle/mobs/Acolnahuacatl.lua

```diff
@@ -9,6 +10,14 @@ entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:addImmunity(xi.immunity.GRAVITY)
+    mob:addImmunity(xi.immunity.BIND)
+    mob:setMod(xi.mod.STORETP, 25)
```

Comment:
Have this store TP balues been confirmed?

If not, could you add a `TODO` comment here and in the other "Store TP" mod down below stating that they need verification?

Reply:
Just went and got a capture. This is going back to a convo in my Rolanberry Fields NM audit where we discussed the store TP on all of these NMs that were added in the Nov 2009 patch. They all have matching bevhaior where they have the same immunties, only use 1 tp move, and have store tp.

https://youtu.be/VrP6haFN8-g

Reply:
I think I didnt word myself properly. I have no doubt it has Store TP. What I am doing is questioning the value of the modifier.

"Store TP" can be, supposedly, calculated with relative ease by some1 that knows how TP works. Like WinterS.
So what I am asking here is if the value is 25 or if the value needs confirmation.

Reply:
Hard to prove exact numbers.  I don't think getting hit indicates the mobs store TP value to the player and NMs are uncharmable by default so you'd have to use something like Dark Seal absorb TP after one hit and then guess and check

Reply:
Thats a good point. I'll have to level up DRK to give that a try, I honestly had no clue how to verify the exact store TP amount.

Reply:
Oh, that jug is way more useful for that since you dont have to wait for dark seal. neat, thanks for that info

Reply:
I've got a BST at 81 for that pet but can't find any of the item to summon. Will try and see if I can find someone on Asura with cooking today, otherwise I'll have to level up DRK for a bit.

Reply:
With Winter's help we were able to get a more accurate store tp value for Acolnahuacatl. I also went back and adjusted the store tp for a few NMs I recently adjusted as well with their more accurate store tp values.